### PR TITLE
Add column with tag name

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix https://github.com/astronomer/issues/issues/477

### CODE REVIEW

File: houston/queries.go [hunk 1]
```diff
@@ -78,6 +78,7 @@ var (
 			version
 			createdAt
 			updatedAt
+			currentTag
```
Comment (ashb) [inline houston/queries.go]: Wait - I just noticed. Isn't this what we ask for in `deployInfo { latest }` already?

**Comment by andriisoldatenko** [comment]
@ashb @schnie PTAL

**Comment by andriisoldatenko** [comment]
```
➜  astro-cli git:(report-current-image-version-to-users) astro deployment list
 NAME     DEPLOYMENT NAME             ASTRO              DEPLOYMENT ID                 TAG
 d1       blazing-ionization-2662     v0.9.0-alpha.4     ck1qg6zv3002f0869ypx5a4ds     cli-2

```

**Comment by schnie** [comment]
Waiting to see what we decide on the Houston PRs before merging this one

**Comment by schnie** [comment]
Wouldn't we want to show the new `currentTag` field?

**Comment by andriisoldatenko** [comment]
@schnie we removed it.

**Comment by andriisoldatenko** [comment]
https://github.com/astronomer/houston-api/pull/147#pullrequestreview-313538778

**Comment by schnie** [comment]
This really only would show people the latest tag that was pushed via the CLI. This wouldn't show them the latest tag if they pushed a custom tag.

@andriisoldatenko can we make an API call to the registry from Houston to get the most recently pushed tag? Or should we keep track in the database?

**Comment by schnie** [comment]
Actually, we can add a `Deployment` resolver for `currentTag`, that grabs it from `DockerImages` right?

**Comment by schnie** [comment]
Or make it an additional field returned from `deployInfo` resolver.

**Comment by andriisoldatenko** [comment]
i think more clean way for me to add extra field to `deployInfo`. @ashb what do you think?

**Comment by andriisoldatenko** [comment]
@schnie 
```
(helm) ➜  astro-cli git:(report-current-image-version-to-users) ✗ go run main.go deployment list
 NAME                    DEPLOYMENT NAME           ASTRO              DEPLOYMENT ID                 TAG
 new-deployment-name     advanced-quark-6540       v0.9.0-alpha.4     ck2gh74fs002h0720940bgdhj     teamcity-r4e3
```

**db:**
```
postgres=# select tag from  "houston$default"."DockerImage";
      tag
---------------
 teamcity-r4e3
(1 row)
```
related PR https://github.com/astronomer/houston-api/pull/164

**Comment by schnie** [comment]
Looks good, but I think there's a naming mismatch in the houston side of this update.

**Pull request merged**